cmake/public/LoadHIP.cmake

```diff
@@ -1,42 +1,86 @@
+set(PYTORCH_FOUND_HIP FALSE)
```

Comment:
Can remove this Variable and rely on HIP_FOUND.

Reply:
See my reply in cmake/public/LoadHIP.cmake